src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const threeDay = 3;
+  const oneWeek = 7;
```

Comment:
```suggestion
  const smallPeriod = 3;
  const bigPeriod = 7;
```
Не привязывай названия переменных к значениям. Если поменяются условия акции, и скидка будет с 4 дней, а не с 3, то тебе нужно будет не только поменять значение в переменной, но и переименовать ее